agent_job_history.go

```diff
@@ -64,3 +54,21 @@ func agentJobHistory(client gocd.Client, agents []*gocd.Agent, agentJobResultCou
 	}
 	return nil
 }
+
+func NewAgentJobHistoryCache() *AgentJobHistoryCache {
+	return &AgentJobHistoryCache{}
+}
+
+func (c *AgentJobHistoryCache) Get(agentHostname string) int {
+	if job, ok := c.AgentJob[agentHostname]; ok {
```

Comment:
this is the same as `return c.AgentJob[agentHostname]`

Reply:
Struct removed